fdbserver/DataDistribution.actor.cpp

```diff
@@ -2569,6 +2578,186 @@ ACTOR Future<Void> waitUntilHealthy(DDTeamCollection* self, double extraDelay =
 	}
 }
 
+// This function is very expensive to call. Use with caution.
```

Comment:
Better comment:
This function takes a snapshot of the team collection details and print them out gracefully.
This function is expensive. Use with caution.

Reply:
Feel free to revise the suggested comment.

Reply:
Done.